pkgs/development/compilers/dotnet/combinePackages.nix

```
@@ -0,0 +1,23 @@
+{ cli
+, withRuntimes ? [] 
+, withSdks ? [] 
+}:
+  assert (builtins.length withRuntimes) != 0 || (builtins.length withSdks) != 0;
```

Better user experience:
```suggestion
  lib.assertMsg ((builtins.length withRuntimes) != 0 || (builtins.length withSdks) != 0) 
    ''You must include at least one runtime or sdk, e.g 
      `with dotnetCorePackages; combinePackages {
          withSdks = [ sdk_3_0 ]; 
          withRuntimes = [ aspnetcore_2_1  ];
       };`'';
  ;
```